features/steps/steps.py

```diff
@@ -34,6 +35,21 @@ def get_inst_attributes(dc):
         yield 'inst_type', dc.inst.is_a()
         yield 'inst_id', dc.inst.id()
 
+def stmt_to_op(statement):
+    statement = statement.replace('is', '').strip()
```

Comment:
Should we also account for `are` here?

Reply:
We could, In the case  below(for which is it used now) it is quite simple. Either the entity is or is not present.
`The X shall be assigned to the Y if Z is present`

But in the case of 'are', I can imagine some complications
`X shall be assigned to Y if all of the following entities are present: A, B, C`
or 
`X shall be assigned to Y if one of the following entities are present:A, B, C `
The second example is similar to the current python implementation of the stmt_to_op function, the first one would be different however.